tools/runners/run-unittests.sh

```diff
@@ -97,7 +105,11 @@ rm -f $UNITTEST_TEMP
 
 ratio=$(echo $passed*100/$total | bc)
 
-echo "[summary] ${DIR#$ROOT_DIR}/unit-*: $passed PASS, $failed FAIL, $total total, $ratio% success"
+echo -e "\n[summary] ${DIR#$ROOT_DIR}/unit-*\n"
+echo -e " TOTAL: $total"
+echo -e "$TERM_GREEN PASS: $passed"
+echo -e "$TERM_RED FAIL: $failed $TERM_EMPTY\n"
```

Comment:
This summary is not tabulated to the same column positions as the summary printed by run-test-suite.sh.